docs/Makefile

```diff
@@ -62,43 +84,23 @@ html: apidoc $(GENERATED_FILES)
 	cp -r $(BUILDDIR)/html/* ..
 
 .PHONY: apidoc
-apidoc: apidoc-animate apidoc-movement apidoc-goalie
+
+apidoc: 
+
+# apidoc-animate apidoc-movement apidoc-goalie
```

Comment:
Do we need this comment?

Reply:
this was an unintentional edit, changed back to the original Makefile format